libtransmission/file-win32.cc

```diff
@@ -911,9 +886,9 @@ bool tr_sys_file_read(tr_sys_file_t handle, void* buffer, uint64_t size, uint64_
     }
 
     bool ret = false;
-    DWORD my_bytes_read;
+    DWORD my_bytes_read = 0;
 
-    if (ReadFile(handle, buffer, (DWORD)size, &my_bytes_read, nullptr))
+    if (to_bool(ReadFile(handle, buffer, (DWORD)size, &my_bytes_read, nullptr)))
```

Comment:
Not sure why this was not caught by clang-tidy...

```suggestion
    if (to_bool(ReadFile(handle, buffer, static_cast<DWORD>(size), &my_bytes_read, nullptr)))
```